calico_containers/tests/st/test_multi_host.py

```diff
@@ -62,18 +49,12 @@ def test_multi_host(self):
         # Wait for the workload networking to converge.
         sleep(1)
 
-        host1.execute("docker exec workload-A ping -c 4 192.168.1.3")
+        assert host1.execute("docker exec workload-A ping -c 4 192.168.1.3")
```

Comment:
I was trying to be all semantically accurate. But since we don't do this anywhere else I'll remove it and wait for an `assert_can_ping` for semantic correctness.